### ISSUE TITLE
Debugger freezes up trying to show a datatip for large Lua tables

### ISSUE THREAD

**Issue by fsfod** [WheretIB/LuaDkmDebugger#3]
The debugger can get bogged down very slowly reading table values in LuaTable.LoadValues when trying to display data tip for large tables with several thousand values. 
This was the call stack it was happening in.
```
	LuaDkmDebuggerComponent.LuaTableData.LoadValues Line 1525	C#	Symbols loaded.
 	LuaDkmDebuggerComponent.EvaluationHelpers.EvaluateValueAtLuaValue Line 224	C#	Symbols loaded.
 	LuaDkmDebuggerComponent.EvaluationHelpers.EvaluateDataAtLuaValue Line 348	C#	Symbols loaded.
 	LuaDkmDebuggerComponent.LocalComponent.Microsoft.VisualStudio.Debugger.ComponentInterfaces.IDkmLanguageExpressionEvaluator.GetItems Line 1649	C#	Symbols loaded.
 	Microsoft.VisualStudio.Debugger.EntryPoint.IDkmLanguageExpressionEvaluator_GetItems	Unknown	No symbols loaded.
```

It would help to have some cut off when reading the values and all those single memory reads could maybe batch read from a single larger memory read thats cached in a byte buffer to speed things up.

**Comment by WheretIB** [comment]
Thank you for the suggestion.

I was able to reproduce the issue and I will try to improve this by using lazy evaluation/caching.

**Comment by WheretIB** [comment]
The original issue was fixed in 86ee12f2c512dbcb269c400a8b7e1477208c3c75

Additional improvements were made for table expression evaluation (process memory read batching, lazy evaluation for node keys and values)

I'm going to test these changes for a bit before I publish the extension update.

**Comment by WheretIB** [comment]
Fix for this issue has been published in 0.9.2 update of the extension.
